cleanlab/count.py

```diff
@@ -224,6 +228,15 @@ def _num_label_issues_multilabel(
     return sum(issues_idx)
 
 
+def _reduce_issues(pred_probs, labels, num_classes):
+    """Returns a boolean mask denoting correct predictions or predictions within a margin around 0.5 for binary classification, suitable for filtering out indices in 'is_label_issue'."""
+    pred = pred_probs.argmax(axis=1)
+    mask = pred == labels
+    if num_classes == 2:
+        mask = mask | ((pred_probs[:, 0] >= 0.5 - EPSILON) & (pred_probs[:, 0] <= 0.5 + EPSILON))
```

Comment:
After thinking more, I think this would be cleaner:

```
pred_probs[:,given_label] += EPS
pred = pred_probs.argmax(axis=1)
mask = pred == labels
```

with this code here, we can get rid of the `if num_classes == 2` statement IMO.

Note that my above code is just pseudocode but you get the idea

Reply:
More formally I believe the line above should be:

```
pred_probs[np.arange(len(labels)), labels] += FLOATING_POINT_COMPARISON
```

Reply:
Added here: https://github.com/cleanlab/cleanlab/blob/309ff7de153573ca030e3c257ded916386e709a6/cleanlab/count.py#L234